src/components/interactive/dragBoxLayer.ts

```diff
@@ -0,0 +1,237 @@
+///<reference path="../../reference.ts" />
+
+module Plottable {
+export module Component {
+  export module Interactive {
+    type _EdgeIndicator = {
+      top: boolean;
+      bottom: boolean;
+      left: boolean;
+      right: boolean;
+    }
+
+    export class DragBoxLayer extends Component.SelectionBoxLayer {
+      private _dragInteraction: Interaction.Drag;
+      private _boxEdgeT: D3.Selection;
+      private _boxEdgeB: D3.Selection;
+      private _boxEdgeL: D3.Selection;
+      private _boxEdgeR: D3.Selection;
+      private _boxCornerTL: D3.Selection;
+      private _boxCornerTR: D3.Selection;
+      private _boxCornerBL: D3.Selection;
+      private _boxCornerBR: D3.Selection;
+      private _boxEdgeWidth = 6;
+
+      private _xResizable = false;
+      private _yResizable = false;
```

Comment:
Just making sure.  So even the new XDragBoxes etc.  will start off as not resizable?